target/linux/kirkwood/base-files/etc/board.d/02_network

```diff
@@ -37,6 +37,9 @@ kirkwood_setup_interfaces()
 	endian,4i-edge-200)
 		ucidef_set_interface_lan "port1 port2 port3 port4 eth1"
 		;;
+	iom,ix4-200d)
+		ucidef_set_interfaces_lan_wan "eth0" "eth1"
```

Comment:
I'm not sure if NAS should be configured like router. Maybe `ucidef_set_interface_lan "eth0 eth1" "dhcp"` will be better?